### PR TITLE
Update docs for removal of Ubuntu 14.04 as a supported platform

### PR DESCRIPTION
**Pull request by hughbe**
@modocache @jrose-apple

### CODE REVIEW

File: docs/Ubuntu14.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# Getting Started with Swift on Ubunutu 14.04
```
Comment (jrose-apple) [inline docs/Ubuntu14.md]: Typo: "Ubunutu".

**Comment by jrose-apple** [comment]
Looks pretty good to me. @tfiala, @shahmishal, what do you think?

**Comment by shahmishal** [comment]
LGTM!

**Comment by modocache** [comment]
@swift-ci please smoke test and merge

**Comment by tfiala** [comment]
LGTM.

**Comment by modocache** [comment]
@swift-ci please smoke test

**Comment by modocache** [comment]
@swift-ci Please smoke test

**Comment by hughbe** [comment]
@modocache is this good to merge?

**Comment by modocache** [comment]
Yup! Sorry for the delay!

**Pull request merged**